dev/com.ibm.ws.jdbc_fat_oracle/test-applications/oraclejdbcfat/src/web/OracleTestServlet.java

```diff
@@ -78,6 +73,10 @@ public class OracleTestServlet extends FATServlet {
     @Test
     @SkipIfSysProp(SkipIfSysProp.OS_IBMI) //Skip on IBM i due to additional Db2 JDBC driver in JDK
```

Comment:
For any of the tests that have `@SkipIfSysProp(SkipIfSysProp.OS_IBMI)` but are not specifically testing DataSource class inference, can we add FIXME to the comment?  That way we can go back and re-enable them with non-generic datasources?

Reply:
Some of these failed when switched to a non-generic datasource, even when run on Windows, like this one in particular. So there would be more to it than just switching.